keps/sig-auth/3257-cluster-trust-bundles/README.md

```diff
@@ -179,6 +180,12 @@ Additionally, this KEP introduces a new `clusterTrustBundle` kubelet projected
 volume source that gives workloads easy filesystem-based access to trust anchor
 sets that they might need.
 
+A new ClusterTrustBundle `kubernetes.io:kube-apiserver-server:live` also gets created
```

Comment:
do we want to commit to a fixed name or just a signer name? I have a slight preference for pods mounting the bundle based on signerName

Reply:
Good point.

Committing just to a signer name sounds good to me. In this case the bundle name should be random so that it's clear that it's not an API. I'll rewrite it to use `generateName: kubernetes.io:kube-apiserver-server:` instead.